ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -36,6 +36,9 @@ library
                      , ouroboros-consensus-mock
                      , ouroboros-consensus-shelley
 
+                     , shelley-spec-ledger
+                     , cardano-crypto-class
+
```

Comment:
Let's put them above `ouroboros-consensus*` deps, together with `cardano-ledger`. See the grouping used for the test-suite in this same cabal file.